src/index.js

```diff
@@ -0,0 +1,21 @@
+import './style.css';
+import enter from './img/enter.png';
+import refresh from './img/refresh.png';
+import dots from './img/dots.png';
+
+document.querySelector('.refresh').src = refresh;
+document.querySelector('.enter').src = enter;
+const contentTask = document.querySelector('.pctasks');
+
+const tasktR = [{ completed: true, description: 'go to run', index: 0 }, { completed: false, description: 'go to eat', index: 2 }, { completed: true, description: 'go to sleep', index: 3 }];
```

Comment:
- [ ] Kindly change the value for completed to false, based on the requirements the value for completed will be false 👍🏿